state/stateenvirons/policy.go

```diff
@@ -39,7 +40,7 @@ type deployChecker interface {
 	environs.ConstraintsChecker
 }
 
-type storageServiceGetter func(modelUUID string, registry storage.ProviderRegistry) state.StoragePoolGetter
+type storageServiceGetter func(modelUUID coremodel.UUID, registry storage.ProviderRegistry) state.StoragePoolGetter
```

Comment:
I probably wouldn't have made this change. This code is going away soon. But fine to leave.

Reply:
ah, wasn't aware